build-tools/packages/build-cli/src/library/releasePrepChecks.ts

```diff
@@ -129,7 +130,7 @@ export const CheckHasRemoteBranchUpToDate: CheckFunction = async (
 
 	let succeeded = false;
 	try {
-		succeeded = await context.gitRepo.isBranchUpToDate(context.originalBranchName, remote);
+		succeeded = await gitRepo.isBranchUpToDate(gitRepo.originalBranchName ?? "", remote);
```

Comment:
Something feels weird here, if we don't have an originalBranchName... should we even be calling `isBranchUpToDate()`?